src/olympia/search/tests/test_search_ranking.py

```diff
@@ -0,0 +1,566 @@
+                'browser fast and clean'),
+            weekly_downloads=85)
+        amo.tests.addon_factory(
+            average_daily_users=26135,
+            description=None,
+            name='Reader',
+            slug=u'reader',
+            summary='Reader is the ultimate Reader tool for Firefox.',
+            weekly_downloads=2463)
+        amo.tests.addon_factory(
+            average_daily_users=53412,
+            description=None,
+            name='Disable WebRTC',
+            slug=u'happy-bonobo-disable-webrtc',
+            summary=(
+                'WebRTC leaks your actual IP addresses from behind your VPN, '
+                'by default.'),
+            weekly_downloads=10583)
+        amo.tests.addon_factory(
+            average_daily_users=12953,
+            description=None,
+            name='In My Pocket',
+            slug=u'in-my-pocket',
+            summary=(
+                'For all those who are missing the old Firefox Pocket addon, '
+                'and not satisfied with the new Pocket integration, here is '
+                'an unofficial client for the excellent Pocket service. '
+                'Hope you\'ll enjoy it!'),
+            weekly_downloads=1123)
+        amo.tests.addon_factory(
+            name='GrApple Yummy')
+        amo.tests.addon_factory(
+            name='Delicious Bookmarks')
+
+        # Some more or less Dummy data to test a few very specific scenarios
+        # e.g for exact name matching
+        amo.tests.addon_factory(
+            name='Merge Windows', type=amo.ADDON_EXTENSION,
+            average_daily_users=0, weekly_downloads=0),
+        amo.tests.addon_factory(
+            name='Merge All Windows', type=amo.ADDON_EXTENSION,
+            average_daily_users=0, weekly_downloads=0),
+        amo.tests.addon_factory(
+            name='All Downloader Professional', type=amo.ADDON_EXTENSION,
+            average_daily_users=0, weekly_downloads=0),
+
+        amo.tests.addon_factory(
+            name='test addon test11', type=amo.ADDON_EXTENSION,
+            average_daily_users=0, weekly_downloads=0),
+        amo.tests.addon_factory(
+            name='test addon test21', type=amo.ADDON_EXTENSION,
+            average_daily_users=0, weekly_downloads=0),
+        amo.tests.addon_factory(
+            name='test addon test31', type=amo.ADDON_EXTENSION,
+            average_daily_users=0, weekly_downloads=0),
+
+        amo.tests.addon_factory(
+            name='1-Click YouTube Video Download',
+            type=amo.ADDON_EXTENSION,
+            average_daily_users=566337, weekly_downloads=150000,
+            description=(
+                'button, click that button, 1-Click Youtube Video '
+                'Downloader is a click click great tool')),
+        amo.tests.addon_factory(
+            name='Amazon 1-Click Lock', type=amo.ADDON_EXTENSION,
+            average_daily_users=50, weekly_downloads=0),
+
+        cls.refresh()
```

Comment:
An improvement for later: prevent the individual index calls while you're adding all these add-ons, then call `reindex` instead of refresh.

Reply:
Good idea, I'll add it to my list of future improvements and will open an issue about it